allennlp/common/util.py

```diff
@@ -47,9 +52,7 @@ def sanitize(x: Any) -> Any:  # pylint: disable=invalid-name,too-many-return-sta
     if isinstance(x, (str, float, int, bool)):
         # x is already serializable
         return x
-    elif isinstance(x, torch.autograd.Variable):
-        return sanitize(x.data)
-    elif isinstance(x, torch._TensorBase):  # pylint: disable=protected-access
+    elif is_tensor(x):
         # tensor needs to be converted to a list (and moved to cpu if necessary)
         return x.cpu().tolist()
```

Comment:
I don't think it would do anything. as I understand it, the main reason to use `.data` is to "break free" of the computational graph; once you convert to a list you're not part of the computational graph anyway